spokestack/tts/clients/spokestack.py

```diff
@@ -65,12 +73,14 @@ def synthesize(
         if "errors" in response:
             raise TTSError(response["errors"])
 
-        response = requests.get(response["data"][_MODES[mode]]["url"], stream=True)
+        response = requests.get(
```

Comment:
this http request should not be made if you are only requesting the audio url
otherwise, the http request will be made twice - once by spokestack and then again by the application
this is why url-based tts should probably just be a separate method